b2/console_tool.py

```diff
@@ -673,10 +673,10 @@ def run(self, args):
 class UploadFile(Command):
     """
     b2 upload_file [--sha1 <sha1sum>] [--contentType <contentType>] [--info <key>=<value>]* \\
-            [--noProgress] [--threads N] <bucketName> <localFilePath> <b2FileName>
+            [--noProgress] [--threads N] [--partSize <partSize>] <bucketName> <localFilePath> <b2FileName>
 
         Uploads one file to the given bucket.  Uploads the contents
-        of the local file, and assigns the given name to the B2 file.
+        of the local file or specify '-' for STDIN, and assigns the given name to the B2 file.
```

Comment:
this doesn't read very well. "Uploads the contents of the local file, and assigns the given name to the B2 file. Specify '-' for to use standard input". Man of cat and bash does not use "STDIN" - we shouldn't either.